src/BigCommerce/Forms/Registration_Handler.php

```diff
@@ -205,10 +193,9 @@ private function validate_submission( $submission ) {
 			$errors->add( 'email', __( 'Email Address is required.', 'bigcommerce' ) );
 		} elseif ( ! is_email( $submission[ 'bc-register' ][ 'email' ] ) ) {
 			$errors->add( 'email', __( 'Please verify that you have submitted a valid email address.', 'bigcommerce' ) );
-		} elseif ( ! $this->is_email_free( $submission['bc-register']['email'] ) ) {
-			$errors->add( 'email', __( 'Sorry, that email address is already used!', 'bigcommerce' ) );
```

Comment:
I mentioned to @bc-donfran about this, but unfortunately there is a string provided by Wordpress that I wasn't able to change. I tried a few different methods, but if someone from the community can help, it would be greatly appreciated here.